runners/flink/src/test/java/org/apache/beam/runners/flink/streaming/UnboundedSourceWrapperTest.java

```diff
@@ -134,11 +133,13 @@ public void testValueEmission() throws Exception {
               public void run() {
                 while (true) {
                   try {
-                    synchronized (testHarness.getCheckpointLock()) {
-                      testHarness.setProcessingTime(System.currentTimeMillis());
-                    }
+                    testHarness.setProcessingTime(System.currentTimeMillis());
                     Thread.sleep(1000);
+                  } catch (InterruptedException e) {
+                    // this is ok
+                    break;
                   } catch (Exception e) {
+                    e.printStackTrace();
```

Comment:
Why not re-throw the exception or log it?

Reply:
It is logged to the screen. Rethrowing in a thread would be possible but it doesn't let the test case fail.

Reply:
I don't consider output to stderr or stdout "logging". That's what we have slf4j and log levels for. Granted it is only test code, but we should remain consistent.

Reply:
+1 If you want to open a PR I'd be happy to ack it!